core/node/events/migrations/snapshot_migration_0002.go

```diff
@@ -0,0 +1,25 @@
+package migrations
+
+import (
+	. "github.com/river-build/river/core/node/protocol"
+	"github.com/river-build/river/core/node/shared"
+)
+
+func snapshot_migration_0002(iSnapshot *Snapshot) *Snapshot {
+	switch snapshot := iSnapshot.Content.(type) {
+	case *Snapshot_SpaceContent:
+		for _, channel := range snapshot.SpaceContent.Channels {
+			if channel.Settings == nil {
+				channel.Settings = &SpacePayload_ChannelSettings{}
+			}
+			channelId, err := shared.StreamIdFromBytes(channel.ChannelId)
+			if err != nil {
+				panic(err)
```

Comment:
i'm a little worried about this panic... and i realize we don't have a context to log anything either... can we just return here instead?

Reply:
I can also just continue and we can try to migrate as many channels as we can

Reply:
panic removed

Reply:
Yes good call, continue is better